### DISCUSSION TITLE
API for querying used in?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#5602] [Q&A] [answered]
I'm trying to find a api that I can call which will tell me what assembly a part is used in.

For example I've for a part number 100 (1k 0805 resistors) which is used in assembly parts 133 and 155. 
On the GUI I can look at part 100 and it will tell me what assemblies it's used in but would like to do this via a API so I can use the data in another database. 

I've had a read through the API docs but don't see anything related to "used in"

Is this hidden in some API?

**Comment by SchrodingersGat** [comment, marked as answer]
I'm away from my computer right now but you could work this out by looking at the "used in" tab on the part page. Use the developer tools in your browser to look at what API calls are being made to load that table.

**Reply by StormTrooper** [reply]
Thanks for the the suggestion. That did the trick. 

API is ```http://inventree.home:8000/api/bom/?search=&uses=100```
